app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/ThreadController.kt

```diff
@@ -242,6 +243,25 @@ class ThreadController @Inject constructor(
             return realm.query<Thread>("${Thread::folderId.name} == $0", folderId)
         }
 
+        private fun getThreadsWithSnoozeFilterQuery(
+            folderId: String,
+            withSnooze: Boolean,
+            realm: TypedRealm,
+        ): RealmQuery<Thread> {
+            val snoozeState = SnoozeState.Snoozed.apiValue
```

Comment:
```suggestion
            val snoozedState = SnoozeState.Snoozed.apiValue
```

Reply:
The name should stay "snooze" and not "snoozed" imo. Not only is it the same name as the api and the other field names, but this way it refers to the "snooze" concept which can be enabled, disabled or whatever, where as "snoozed" would refer more to the "enabled snooze" concept

Reply:
Ok it's used only to hold the "snoozed" value. Understandable. Actually I dont' even need the intermediate variable